### PR TITLE
global: beta release preparation

### PR DESCRIPTION
**Pull request by slint**

### CODE REVIEW

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -95,7 +95,12 @@ Ready to contribute? Here's how to set up `invenio` for local development.
    .. code-block:: console
 
       $ git add .
-      $ git commit -s -m "Your detailed description of your changes."
+      $ git commit -s
+          -m "component: title without verbs"
+          -m "* NEW Adds your new feature."
+          -m "* FIX Fixes an existing issue."
+          -m "* BETTER Improves and existing feature."
+          -m "* Changes something that should not be visible in release notes"
```
Comment (jirikuncar) [inline CONTRIBUTING.rst]: missing dot :wink:

**Comment by lnielsen** [comment]
Ready from our side. Fixed the dot and ensured test cover age 100%

**Comment by jirikuncar** [comment]
Coveralls is not responding. Otherwise LGTM

**Comment by lnielsen** [comment]
LGTM. Coveralls responded now, just LGTM that needs to report status.

**Comment by jirikuncar** [comment]
LGTM

**Pull request merged**